.github/file-filters.yml

```diff
@@ -50,16 +46,21 @@ backend_build_changes: &backend_build_changes
   - 'Makefile'
   - 'pyproject.toml'
 
-backend: &backend
+backend_no_tests: &backend_no_tests
   - *backend_build_changes
-  - *backend_lintable
+  - *backend_dependencies
+  - '!(tests)/**/*.py'
```

Comment:
just to confirm -- this matches `src/foo.py` but not `tests/src/foo.py` ? what happens if both are present?

Reply:
When you edit...
- `src/foo.py`
    - backend = true